src/r_brushmodel_sky.c

```diff
@@ -208,7 +232,7 @@ static qbool Sky_LoadSkyboxTextures(const char* skyname)
 				fixed_size = size;
 
 				if (i == 0) {
-					skybox_cubeMap = R_CreateCubeMap("***skybox***", size, size, TEX_NOCOMPRESS | TEX_MIPMAP | TEX_NOSCALE);
+					skybox_cubeMap = R_CreateCubeMap("***skybox***", size, size, TEX_NOCOMPRESS | TEX_MIPMAP | TEX_NOSCALE | TEX_ALPHA);
```

Comment:
This is mostly the only impact of this feature. Skyboxes will be 4bpp instead of 3bpp going forward. Uses more GPU memory, but people who care about GPU memory likely disable skyboxes, and it's not like it's a huge difference.